src/ess/polarization/he3.py

```diff
@@ -209,23 +251,38 @@ def get_he3_transmission_from_fit_to_direct_beam(
     def expected_transmission(
         wavelength: sc.Variable, time: sc.Variable, C: sc.Variable, T1: sc.Variable
     ) -> sc.Variable:
-        opacity = opacity_function(wavelength)
         polarization_function = He3PolarizationFunction[Cell](C=C, T1=T1)
-        polarization = polarization_function(time)
-        return transmission_incoming_unpolarized(
-            transmission_empty_glass=transmission_empty_glass,
-            opacity=opacity,
-            polarization=polarization,
+        if incoming_polarized:
+            # TODO Why is this 'plus'? Does it depend on the supermirror?
```

Comment:
@astellhorn Can you comment on this? Does it depend on which period we load, i.e., should this be made configurable?